mmdet/models/utils/inverted_residual.py

```diff
@@ -119,6 +124,9 @@ def _inner_forward(x):
 
             if self.with_res_shortcut:
                 return x + self.drop_path(out)
+
+            if self.return_list:
+                return [expanded_out, out]
```

Comment:
self.with_res_shortcut may shortcut self.return_list branch.

Reply:
Yes.. I think I should fix it for generality. For now, self.with_res_shortcut is always False when self.return_list is True because the stride value is 2.